content/en/_index.html

```diff
@@ -43,12 +43,12 @@ <h2>The Challenges of Migrating 150+ Microservices to Kubernetes</h2>
         <button id="desktopShowVideoButton" onclick="kub.showVideo()">Watch Video</button>
         <br>
         <br>
-        <a href="https://events.linuxfoundation.org/kubecon-cloudnativecon-north-america/?utm_source=kubernetes.io&utm_medium=nav&utm_campaign=kccncna20" button id="desktopKCButton">Attend KubeCon NA virtually on November 17-20, 2020</a>
+        <a href="https://events.linuxfoundation.org/kubecon-cloudnativecon-north-america/?utm_source=kubernetes.io&utm_medium=nav&utm_campaign=kccncna21" button id="desktopKCButton">Attend KubeCon in Los Angeles on October 12-15, 2021</a>
```

Comment:
Hi @jihoon-seo .
Should this announcement include "NA"?
Kubecon NA in Los Angeles, CA?

/cc @celestehorgan

Reply:
@kbhawkey 

Answer from marketing: "KubeCon + CloudNativeCon North America" is the full name that should be used :) 

If you can't fit all ofthat on the button (which... I suspect you may be unable to), "KubeCon North America" will do.

Reply:
Thanks for the information @celestehorgan !

@jihoon-seo , Can you update the announcement with the suggested text?
I think "KubeCon North America" would fit?
Is it essential to include the location?